pkg/apis/eventbus/v1alpha1/kafka_eventbus.go

```diff
@@ -0,0 +1,44 @@
+package v1alpha1
+
+import (
+	apicommon "github.com/argoproj/argo-events/pkg/apis/common"
+	corev1 "k8s.io/api/core/v1"
+)
+
+// KafkaBus holds the KafkaBus EventBus information
+type KafkaBus struct {
+	// Exotic holds an exotic Kafka config
+	Exotic *KafkaConfig `json:"exotic,omitempty" protobuf:"bytes,1,opt,name=exotic"`
+}
+
+type KfConsumerGroup struct {
```

Comment:
Is it possible to name this `KafkaConsumerGroup`? Also can we define this below `KafkaConfig` as this is the order they are used.